src/bullets/data_source/data_source_fmp.py

```diff
@@ -12,40 +14,96 @@ def __init__(self, token: str, resolution: Resolution):
         super().__init__()
         self.token = token
         self.resolution = resolution
+        self.stocks = {}
+
+    def store_price_points(self, symbol: str, start_date: datetime.date, end_date: datetime.date = None,
```

Comment:
Are people coding their strategies supposed to call this method? If not, you can mark it as private by adding an underscore at the beggining (`_store_price_points`)